elements/d-d-d/package.json

```diff
@@ -44,13 +44,25 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@haxtheweb/hax-iconset": "^9.0.22",
-    "@haxtheweb/simple-colors": "^9.0.22",
-    "@haxtheweb/simple-icon": "^9.0.22",
```

Comment:
stop changing this file this is not possible to be fixed as it stands presently. There is now a d-d-docs element that hasn't shown up yet and should work